seatunnel-e2e/seatunnel-connector-v2-e2e/connector-starrocks-e2e/src/test/resources/starrocks-to-starrocks.conf

```diff
@@ -45,4 +62,7 @@ sink {
         sink.properties.strip_outer_array = true
         max_retries = 3
     }
+    Console {
+
+    }
```

Comment:
Add `Assert` Sink check rows & fields

Reply:
> Add `Assert` Sink check rows & fields

thanks，StarRocksIT contains Sink check rows & fields

Reply:
added Benchmarking（starRocks thrift source connector) in commente